resources/other/userAgents.json

```diff
@@ -22,8 +27,8 @@
 	},
 	{
 		"identifier": "internet-explorer",
-		"title": "Mask as Internet Explorer 10.0 (Windows)",
-		"value": "Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; Trident/6.0)"
+		"title": "Mask as Internet Explorer 11.0 (Windows)",
+		"value": "Mozilla/5.0 (compatible; MSIE 11.0; Windows NT 6.1; Trident/7.0)"
```

Comment:
I don't think that's a thing?

https://msdn.microsoft.com/en-us/library/bg182625(v=vs.85).aspx#uaString